dpar/src/models/tensorflow/collector.rs

```diff
@@ -108,7 +139,15 @@ where
 
         self.vectorizer.realize_into(
             state,
-            &mut self.inputs[batch].to_instance_slices(self.instance_idx),
+            &mut self.lookup_inputs[batch].to_instance_slices(self.instance_idx),
+            &mut self.non_lookup_inputs[batch][(self.instance_idx
```

Comment:
Factor out the shared parts of the range bounds to a varuable. (Something like `non_lookup_inputs_offset`.)